### ISSUE TITLE
Nugets not on preview

### ISSUE THREAD

**Issue by NunoPeixotoPrimavera** [microsoft/appcenter#1347]
When the nugets will be officially out of preview? (And also strong name signed? - this is a very common request!)

**Comment by elamalani** [comment]
@NunoPeixotoPrimavera Are you talking about Auth and Data NuGets?

**Comment by amchew** [comment]
Hey there @NunoPeixotoPrimavera! Following up on this, are you talking about Auth and Data NuGets? If so, there’s no ETA yet on when we’ll remove the preview tag for Auth and Data.

**Comment by amchew** [comment]
Hey there @NunoPeixotoPrimavera, do you have any other questions?

**Comment by Zakeelm** [comment]
@NunoPeixotoPrimavera  Thank you so much for your request. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data service later this year and recommending customers transition to  Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
